go.work

```diff
@@ -1,4 +1,4 @@
-go 1.21
+go 1.23
```

Comment:
There is no need to update the go.mod go version, it can and therefore should stay on 1.21.

Use the below statement to determine if it needs to, as it will fail if it does

```
go mod tidy -go=1.21 -compat=1.21
```